sdk/core/azure-core/src/http/curl/curl_session_private.hpp

```diff
@@ -258,7 +258,8 @@ namespace Azure { namespace Core { namespace Http {
      * provide their own buffer to copy from socket when reading the HTTP body using streams.
      *
      */
-    uint8_t m_readBuffer[Details::c_DefaultLibcurlReaderSize]; // to work with libcurl custom read.
+    uint8_t m_readBuffer[Details::c_DefaultLibcurlReaderSize]
+        = {}; // to work with libcurl custom read.
```

Comment:
Is this 0-initializing all the elements of the array? @CaseyCarter what's the fastest/best approach to initialize an array member?

Reply:
Yes, this effectively initializes all elements of the array to `0`.

Reply:
Let's be explicit then, and init all to `0`.

```suggestion
    uint8_t m_readBuffer[Details::c_DefaultLibcurlReaderSize]
        = { 0 }; // to work with libcurl custom read.
```